### PR TITLE
Adds Randy's bio

### PR DESCRIPTION
**Pull request by gboone**
cc: @randyhart @18F/blog @18F/content

### CODE REVIEW

File: _team/randy-hart.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+full_name: Randy Hart
+github:
+last_name: Hart
+name: randy-hart
+role: Procurement specialist
+state: DC
+team: Consulting
+twitter:
+---
+Randy is a procurement specialist. He joined the government in 2002 and worked as a Contracting Officer for the Census Bureau and the Navy. As a "Level 3" officer, he was trained to compete and negotiate contracts worth more than $100 million. "I've bought everything from software and hardware to large integrated systems," he said of his work before 18F.
```
Comment (emileighoutlaw) [inline _team/randy-hart.md]: So here in the body, I think Contracting Officer should _not_ be capitalized.

File: _team/randy-hart.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+city: Washington
+first_name: Randy
+full_name: Randy Hart
+github:
+last_name: Hart
+name: randy-hart
+role: Procurement specialist
```
Comment (emileighoutlaw) [inline _team/randy-hart.md]: Procurement Specialist (in keeping with how we've been doing it on other pages).

File: _team/randy-hart.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+state: DC
```
Comment (emileighoutlaw) [inline _team/randy-hart.md]: D.C.

File: _team/randy-hart.md (hunk 2 above)
Comment (gboone) [inline _team/randy-hart.md]: Yeah, I don't even know if that's the right title. Will need Randy or someone from consulting to weigh in on that.

**Pull request merged**